src/data/__snapshots__/redux.test.jsx.snap

```diff
@@ -4,21 +4,23 @@ exports[`Data layer integration tests Test exams IDA url Should call the exams s
 Object {
   "examState": Object {
     "activeAttempt": Object {
-      "accessibility_time_string": "you have 30 minutes remaining",
-      "attempt_code": "",
-      "attempt_id": 1,
-      "attempt_status": "started",
-      "course_id": "course-v1:test+special+exam",
-      "desktop_application_js_url": "",
-      "exam_display_name": "timed",
-      "exam_started_poll_url": "/api/edx_proctoring/v1/proctored_exam/attempt/1",
-      "exam_type": "a timed exam",
-      "exam_url_path": "http://localhost:2000/course/course-v1:test+special+exam/block-v1:test+special+exam+type@sequential+block@abc123",
-      "in_timed_exam": true,
-      "software_download_url": "",
-      "taking_as_proctored": false,
-      "time_remaining_seconds": 1799.9,
-      "total_time": "30 minutes",
+      "attempt": Object {
```

Comment:
This shouldn't be a nested dictionary, the snapshot for this test was correct before. If you look at the other existing tests, you'll see that the right shape is {activeAttempt: ...}, not {activeAttempt: {attempt: ...}}